clients/hydrophone/hydrophone_client.go

```diff
@@ -114,3 +139,30 @@ func (client *Client) GetPendingInvitations(userID string, authToken string) ([]
 		}
 	}
 }
+
+func (client *Client) CancelSignup(userID string, authToken string) error {
+	host, err := client.getHost()
+	if err != nil {
+		return errors.New("No known hydrophone hosts")
+	}
+
+	host.Path = path.Join(host.Path, "signup", userID)
```

Comment:
```suggestion
	host.Path = path.Join(host.Path, "dismiss/signup", userID)
```

Reply:
No, dismiss is a different API ... a PUT on signup is doing a cancel ;) weird but it's the API today ...